lib/rules/no-unused-placeholders.js

```diff
@@ -57,7 +57,7 @@ module.exports = {
           ) {
             const message = reportInfo.message.value || messageStaticValue.value;
             // https://github.com/eslint/eslint/blob/2874d75ed8decf363006db25aac2d5f8991bd969/lib/linter.js#L986
-            const PLACEHOLDER_MATCHER = /\{\{\s*([^{}]+?)\s*\}\}/g;
+            const PLACEHOLDER_MATCHER = /{{\s*([^{}]+?)\s*}}/g;
```

Comment:
a false negative of the rule [no-useless-escape](https://eslint.org/docs/rules/no-useless-escape)?

Reply:
Looks like it, [unicorn/better-regex](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/better-regex.md) applies a few more plugins to cleanup the regexp.

Reply:
can you file the issue in eslint repo, to make sure we don't lost track of it? thanks!